src/lumigo_tracer/spans_container.py

```diff
@@ -87,6 +89,7 @@ def __init__(
                     "logGroupName": log_group_name,
                     **(trigger_by or {}),
                 },
+                "tags": [],
```

Comment:
note that we send it also in the `_started` span

Reply:
Ok, do you in can cause a problem?

Reply:
It depends on you BE impl., so be careful :)